target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -9,16 +9,16 @@ mediatek_setup_interfaces()
 
 	case $board in
 	asus,tuf-ax4200)
-		CI_UBIPART="UBI_DEV"
```

Comment:
Any idea why this was there from the start?

Reply:
```
	asus,tuf-ax4200)
		CI_UBIPART="UBI_DEV"
		addr=$(mtd_get_mac_binary_ubi "Factory" 0x4)
```
I think this is just a simple copy from other places.